src/services/K8sCleanupService.ts

```diff
@@ -0,0 +1,298 @@
+    }
+
+    /**
+     * Get deployed secrets
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getDeployedSecrets(): Promise<string[]> {
+        return this.getDeployedK8sObjects()
+            .filter(o => o.kind === 'Secret')
+            .map(o => o.metadata.name)
+        ;
+    }
+
+    /**
+     * Get deployed config maps
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getDeployedConfigMaps(): Promise<string[]> {
+        return this.getDeployedK8sObjects()
+            .filter(o => o.kind === 'ConfigMap')
+            .map(o => o.metadata.name)
+        ;
+    }
+
+    /**
+     * Get deployed storage classes
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getDeployedStorageClasses(): Promise<string[]> {
+        return this.getDeployedK8sObjects()
+            .filter(o => o.kind === 'StorageClass')
+            .map(o => o.metadata.name)
+        ;
+    }
+
+    /**
+     * Get deployed persistent volume claims
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getDeployedPersistentVolumeClaims(): Promise<string[]> {
+        return this.getDeployedK8sObjects()
+            .filter(o => o.kind === 'PersistentVolumeClaim')
+            .map(o => o.metadata.name)
+        ;
+    }
+
+    /**
+     * Get installed helms in the cluster
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getClusterHelms(): Promise<string[]> {
+        return await Container.get(K8sHelmService).listInstalledHelms() || [];
+    }
+
+    /**
+     * Get installed secrets in the cluster
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getClusterSecrets(): Promise<string[]> {
+        return await Container.get(K8sKubectlService).listObjects('Secret', this.options.namespace);
+    }
+
+    /**
+     * Get installed config maps in the cluster
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getClusterConfigMaps(): Promise<string[]> {
+        return await Container.get(K8sKubectlService).listObjects('ConfigMap', this.options.namespace);
+    }
+
+
+    /**
+     * Get installed storage classes in the cluster
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getClusterStorageClasses(): Promise<string[]> {
+        return await Container.get(K8sKubectlService).listObjects('StorageClass', this.options.namespace);
+    }
+
+    /**
+     * Get installed persistent volume claim in the cluster
+     *
+     * @return {Promise<string[]>}
+     */
+    private async getClusterPersistentVolumeClaims(): Promise<string[]> {
+        return await Container.get(K8sKubectlService).listObjects('PersistentVolumeClaim', this.options.namespace);
+    }
+
+    /**
+     * Return Difference
+     *
+     * @param {string[]} inCluster
+     * @param {string[]} deployed
+     * @return {string[]}
+     */
+    private getDifferenceInstalled(inCluster: string[], deployed: string[]): string[] {
```

Comment:
`findOrphans` - difference between all objects inside the cluster and ones that were deployed